clients/src/main/java/org/apache/kafka/clients/producer/KafkaProducer.java

```diff
@@ -458,7 +458,6 @@ Sender newSender(LogContext logContext, KafkaClient kafkaClient, ProducerMetadat
                 apiVersions,
                 throttleTimeSensor,
                 logContext);
-        int retries = configureRetries(producerConfig, log);
```

Comment:
Then we don't need `configureRetries` anymore, right?

Reply:
oops. 👍